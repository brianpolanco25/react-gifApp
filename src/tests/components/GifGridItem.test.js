
import React from 'react';
import { shallow } from 'enzyme';
import '@testing-library/jest-dom';
import { GifGridItem } from '../../components/GifGridItem';

describe('Pruebas en el componente <GifGridItem />', () => {

    const title = 'Is a title';
    const url = 'https://localhost.jpg';

    test('Debe cargar correctamente el componente', () => {
        
        const wrapper = shallow(<GifGridItem title = { title } url={ url } />);
        expect(wrapper).toMatchSnapshot();
    })
    
})
