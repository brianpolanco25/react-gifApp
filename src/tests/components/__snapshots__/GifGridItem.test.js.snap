// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente <GifGridItem /> Debe cargar correctamente el componente 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="Is a title"
    src="https://localhost.jpg"
  />
  <p>
    Is a title
  </p>
</div>
`;
